utils/README.md

```diff
@@ -0,0 +1,50 @@
+# APAeval utils
+
+This directory provides Dockerfiles and corresponding 'recipes' for scripting/processing tasks that are common across workflows.
+
+Contents:
+- [Available utilities](#Available-utilities)
+- [Contributing instructions](#Contributing-instructions)
+
+# Available utilities
+
+## Convert GTF to BED12 (aka 'gene model') file
+
+**Description:** Convert a [GTF](https://genome.ucsc.edu/FAQ/FAQformat.html#format4) annotation file to [BED12](https://genome.ucsc.edu/FAQ/FAQformat.html#format1) (also known as a 'gene model' file) format.
+
+**subdirectory name**: **{update this}**
+
+**Dockerhub link:** https://hub.docker.com/r/apaeval/gtftobed
+
+**Dockerhub url (for pipeline)**: `docker.io/apaeval/gtftobed:1.0`
+
+**Usage (shell command to run the tool)**:
+```
+python3 /app/gtfTobed.py --gtf <path to GTF file>
```

Comment:
See comment at Dockerfile; can `gtfTobed.py` be made executable within the container, such that this call can be changed to `python3 gtfTobed.py --gtf...`?